src/utilities/table/table.ts

```diff
@@ -74,7 +74,7 @@ export function numericFiltering(columnData: Array<number>, dataIndexes: number[
         return [];
     }
 
-    let compareFunction = undefined;
+    let compareFunction: (data: number) => boolean;
```

Comment:
If the function is undefined originally, does it mean we should add | undefined to prevent regression?

Reply:
Yes, this looks safer. 👍